assets/app/scripts/directives/metrics.js

```diff
@@ -0,0 +1,257 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .directive('podMetrics', function($interval, $parse, MetricsService, usageValueFilter) {
+    return {
+      restrict: 'E',
+      scope: {
+        pod: '=',
+        alerts: '='
+      },
+      templateUrl: 'views/directives/_pod-metrics.html',
+      link: function(scope) {
+        var intervalPromise;
+        var getMemoryLimit = $parse('resources.limits.memory');
+        var getCPULimit = $parse('resources.limits.cpu');
+
+        function bytesToMB(value) {
```

Comment:
are we going to want to convert to GB in some cases?  Also I think its actually MiB not MB that the resource limit is stored as. At least that is the case on the limit range object.  @derekwaynecarr can you confirm what units memory is in on the pod/container.